assets/js/components/alert.tsx

```diff
@@ -0,0 +1,18 @@
+import { h, ComponentChildren } from "preact";
+
+export function Alert({
+  heading,
+  children,
+}: {
+  heading?: string;
```

Comment:
It existed previously, and we're not currently leveraging it's optional-ness, but I'm thinking we should either make this required, or handle it in the render logic so that we don't output the `<h5>` if it's empty, in order to avoid (1) weird visual offset and (2) [accessibility issue](https://dequeuniversity.com/rules/axe/3.5/empty-heading?application=AxeChrome). That said, ideally we aren't implementing these components in each project, either having one source of design system components, or using something like [react-uswds](https://github.com/trussworks/react-uswds).

![image](https://user-images.githubusercontent.com/1779930/160418134-5002d850-6157-423d-818e-19c8b3ae6fc9.png)

```suggestion
  heading: string;
```

Reply:
updated component to avoid the empty tag in d56d337153c68f75b078bd4d374afe14a2eca2c5